### DISCUSSION TITLE
Camera default parameters

### DISCUSSION THREAD

**Discussion by mshooter** [scripthookvdotnet/scripthookvdotnet#1298] [Q&A] [answered]
Hi everyone, 

I generated data a while ago using script hook. 
But I did not safe the near/far clipping plane of the camera.
Could anyone tell me what the default values are in the game ? 

Thank you

**Comment by kagikn** [comment, marked as answer]
For "default_scripted_camera" (for the `camScriptedCamera` class), the near clip is set to 0.15 and the far clip is set to 800 on creation. For other types of `camBaseCamera` classes, I haven't tested.

**Reply by mshooter** [reply]
Thank you! Is this in meters?

**Reply by kagikn** [reply]
In meters, yes. This also applies to depth of field stuff. Well, GTA uses the metric system and the distance between 2 coordinates can be calculated in meters without additional multiplication.

**Reply by mshooter** [reply]
Thank you :)
